pkgs/development/python-modules/scipy/default.nix

```diff
@@ -100,6 +104,12 @@ buildPythonPackage {
     pkg-config
     wheel
     setuptools
+    # A shim xcrun executable, to fool upstream's meson.build that we have or
+    # on a macos with new enough SDK. We don't have the real executable
+    # available as a real package in Nixpkgs.
+    (writeShellScriptBin "xcrun" ''
+      echo "13.3"
+    '')
```

Comment:
Why is this not conditioned by `stdenv.isDarwin`?

Reply:
> Why is this not conditioned by `stdenv.isDarwin`?

Hmm I was thinking about it, but it doesn't hurt at all to put it in the Linux build as well, as this executable is not even being run on a Linux build. If you find a good choice to condition this I won't mind to, it will make the diff a bit more ugly due to the `nixfmt` requirement...

Reply:
It felt a bit weird to have it on linux.  And the thousands of linux rebuilds are also a consideration at this point.

Reply:
> It felt a bit weird to have it on linux. And the thousands of linux rebuilds are also a consideration at this point.

That's a good reason. I also fixed the `pythonOlder` commit such that it won't trigger linux rebuilds.